src/ui/flutter_app/lib/mill/rule.dart

```diff
@@ -18,7 +18,7 @@
 
 import 'package:sanmill/services/language_info.dart';
 
-// TODO: deprecate this thingy. No reason to keep it
+// TODO: [Leptopoda] deprecate this thingy. No reason to keep it
```

Comment:
I don't think our internal comments need to follow a strict style guide (also this particular one is removed in a future commit)
The same below

Reply:
The to-do items have priority. If one day in the future, the waste needs to be processed in a unified manner, it may be missed during the full text search.  However, temporary comments do not have much effect.  As long as you can control it.

Reply:
BTW. In fact, the spell check plugin of VSCode is easy to use. When the wavy line appears, just select the suggested word from the drop-down list.

The speeling ignore list has been added to VSCode on the branch dev, so it should be easy to use this plugin now without being disturbed by the wavy lines that can be seen everywhere.